README.md

```diff
@@ -9,6 +9,7 @@ Standard Mac setup for the team at Studio 24.
 * [Mac setup for all staff](mac-setup.md)
 * [Telephone setup](telephone.md)
 * [Accessing VPN](vpn.md)
+* [Apple ID](appleid.md)
```

Comment:
to follow other file naming can you rename to: apple-id.md

Reply:
Updated